src/main/java/tc/oc/pgm/spawns/states/Observing.java

```diff
@@ -109,4 +111,13 @@ public void onEvent(EntityDamageEvent event) {
     super.onEvent(event);
     event.setCancelled(true);
   }
+
+  @Override
+  public void onEvent(PlayerInteractEvent event) {
+    super.onEvent(event);
+    if (event.getItem().getType() == Material.LEATHER_HELMET) {
```

Comment:
Is there a static variable for the helmet join hat? Because this will prevent *any* leather hat.